multi-node-test.sh

```diff
@@ -89,16 +89,15 @@ while [ $i -gt 0 ]; do
     sudo rmdir $CART_BASE || find $CART_BASE || true
     if [ \$rc != 0 ]; then
         exit \$rc
-    fi" 2>&1 | dshbak -c
-    if [ ${PIPESTATUS[0]} = 0 ]; then
+    fi"; then 
         i=0
     fi
     let i-=1
 done' EXIT
 
 CART_BASE="${SL_PREFIX%/install/*}"
-if ! pdsh -l jenkins -R ssh -S \
-          -w "$(IFS=','; echo "${nodes[*]:0:$1}")" "set -ex
+if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
+    -w "$(IFS=','; echo "${nodes[*]:0:${1%%-*}")" "set -ex
```

Comment:
(lint) The mentioned parser error was in this double quoted string. [SC1009]
(lint) Couldn't parse this parameter expansion. [SC1073]